src/AutoRest.CSharp/Common/Input/Source/SourceTypeMapping.cs

```diff
@@ -6,29 +6,36 @@
 
 namespace AutoRest.CSharp.Input.Source
 {
-    public class ModelTypeMapping
+    public class SourceTypeMapping
```

Comment:
I rename this type to `SourceTypeMapping` because it does not only apply to model